pkg/vere/main.c

```diff
@@ -2128,123 +2131,192 @@ _cw_chop(c3_i argc, c3_c* argv[])
   }
 
   // gracefully shutdown the pier if it's running
-  u3_disk* old_u = _cw_disk_init(u3_Host.dir_c);
+  u3_disk* log_u = _cw_disk_init(u3_Host.dir_c);
 
-  // note: this include patch applications (if any)
+  // note: this includes patch applications (if any)
   u3m_boot(u3_Host.dir_c, (size_t)1 << u3_Host.ops_u.lom_y);
 
   // check if there's a *current* snapshot
-  if ( old_u->dun_d != u3A->eve_d ) {
+  if ( log_u->dun_d != u3A->eve_d ) {
     fprintf(stderr, "chop: error: snapshot is out of date, please "
                     "start/shutdown your pier gracefully first\r\n");
-    fprintf(stderr, "chop: eve_d: %" PRIu64 ", dun_d: %" PRIu64 "\r\n", u3A->eve_d, old_u->dun_d);
+    fprintf(stderr, "chop: eve_d: %" PRIu64 ", dun_d: %" PRIu64 "\r\n", u3A->eve_d, log_u->dun_d);
```

Comment:
NIT: there are a lot of these so it'd probably be easier to fix up at the end. `eve_d`, `dun_d`, etc. are /aliases/ to fixed width 64 bit integers. You shouldn't use the 64 print specifier directly. I added corresponding print specifiers to types.h to use with c3 types. These are convenient because you don't have to think about integer width at all.

Use PRIc3_d for these since they're a c3_d type.

Reply:
Great thank you. Done.